Dockerfile

```diff
@@ -0,0 +1,37 @@
+# Build stage
+FROM python:3.9-slim AS build
+
+# Set environment variables
+ENV PYTHONUNBUFFERED=1
+
+# Set work directory
+WORKDIR /app
+
+# Copy requirements and install dependencies
+COPY requirements.txt .
+RUN pip install --no-cache-dir -r requirements.txt
+
+# Copy project files
+COPY . .
+
+# Run stage
+FROM python:3.9-slim AS run

```

Comment:
The alias 'run' is correctly used for the run stage. Ensure that the syntax 'AS run' is supported by your Docker version.